fc/cli/watch.py

```diff
@@ -0,0 +1,27 @@
+import click
+from ..jira.issue import Issue
+from ..auth.combo import ComboAuth
+from . import cli_library
+
+
+@click.command()
+@click.option('--username')
+@click.argument('issue_ids', nargs=-1, metavar='ISSUE-1 ISSUE-2 ISSUE-3 etc.')
+def watch(username: str, issue_ids: tuple):
+    auth = ComboAuth(username)
+
+    for i in issue_ids:
+        click.echo("Adding '{}' to the watchlist for {}".format(auth.username(), i))
+
+        the_issue = None
+
+        try:
+            the_issue = Issue.get_issue(i, auth)
+        except Exception as e:
+            cli_library.fail_execution(1, 'Issue retrieval failed with {}'.format(e))
+
+        if the_issue is not None:
```

Comment:
I think it is impossible for the issue to be `None`.  _Something_ will come back.  But what we should catch is an `HTTPError` exception.

Reply:
OK,  I will fix that up then.

Reply:
So once I moved it to task, I renamed stuff to the `the_task` I noticed I have to import `HTTPError` from requests as it is wrapped from urllib3. Do you want me to stick with replacing `if the_issue is not None:` with specifically an `HTTPError` and import that from requests into task.py or just use a generic `Exception` catch. I settled on the latter for now.

Reply:
Yeah, I think importing and using `HTTPError` in `task.py` is fine.

Reply:
OK,  done.